### DISCUSSION TITLE
Video Recording

### DISCUSSION THREAD

**Discussion by sametkarakurt** [react-native-image-picker/react-native-image-picker#1988] [Q&A]
Does the video recording feature work? 
I tried this options 
 const options = {
    title: 'Take Video',
    type: 'capture',
    options: {
      saveToPhotos: true,
      mediaType: 'video',
    }, 
  }

But it opened camera to take photo.

**Comment by EDible-Dev** [comment]
Yes the video recording feature works.  We did it with:

`        const result = await launchCamera({ mediaType: 'video', saveToPhotos: true });
`

Hope that helps
